Intersect.Editor/Content/ContentManager.cs

```diff
@@ -524,7 +524,7 @@ public static object GetSound(string name)
 
         public static string[] GetSmartSortedTextureNames(TextureType type)
         {
-            return SmartSort(GetTextureNames(type));
+            return GetTextureNames(type).Length > 0 ? SmartSort(GetTextureNames(type)) : new[] {string.Empty};
```

Comment:
Big no-no.

You should save the result of `GetTextureNames(type)` to a local variable, otherwise you're calling a function twice, which could return a different value each time.

Reply:
Thanks for reviewing, will do 👍🏽